bucket/paperwork.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "2.0",
+    "description": "Personal document manager",
+    "homepage": "https://openpaper.work",
+    "license": "GPL-3.0-only",
+    "url": "https://download.openpaper.work/windows/x86/paperwork-master-24ba2639.zip",
+    "hash": "85414f4789e000b60ede495df8e314c7328d47ae16a163c1110e0d1a0b9cbdcd",
+    "shortcuts": [
+        [
+            "paperwork.exe",
+            "Paperwork",
+            "",
+            "data/paperwork_64.ico"
+        ]
+    ],
+    "checkver": {
+        "url": "https://gitlab.gnome.org/api/v4/projects/World%2FOpenPaperwork%2Fpaperwork/repository/tags?order_by=name",
```

Comment:
![A](https://i.imgur.com/pLRx11j.png)

Go through the source code/reverse engineer and provide native version check.

Reply:
https://openpaper.work/beacon/latest is used for the update check. This gives me the version but for the download URL I need the commit hash.